### ISSUE TITLE
Feedback Ex 6: #20

### ISSUE THREAD

**Issue by kristianolesenlarsen** [Kristianuruplarsen/sdseml_online_exercises#92]
@daconom (please reply to this issue when you have seen the feedback)

**The warmup**
The sum looks super. I have a simpler version of the bonus, which you can see in the solution slides, but yours work too. Consider using return instead of print, even though this specific case is "print-y", using print as a replacement for function returns is a bad habit that will surely confuse you at some point down the line. 

**The exercises**
Your solutions look good! It is a good idea to use a dictionary to keep track of the return values from `simulate`. In the last question you get a mistake that is understandably confusing. Essentially you run into issues with the _namespace_. Note that I import `sklearn.linear_model.Lasso` above the `PostLasso` definition. Now python associates the name `Lasso` with the class stored in `sklearn.linear_model.Lasso`. Then in the final codeblock you define 
```python
Lasso = PostLasso()
```
so now python no longer associates `Lasso` with `sklearn.linear_model.Lasso`, but with an instance `PostLasso()`. But `PostLasso` relies on the original association in it's `__init__` method, so the second time you run the codeblock it fails and gives you 
```
'PostLasso' object is not callable
```
because `Lasso=PostLasso()` and you cannot call `PostLasso()()`. In short your error goes away if you rename `Lasso = PostLasso()` to e.g. `post_lasso = PostLasso()`.

**Score: 1**

**Comment by daconom** [comment]
Thanks for the detailed response!
